packages/api/tests/lambdas/test-sf-starter.js

```diff
@@ -320,3 +325,21 @@ test('handleThrottledEvent respects maximum executions for multiple priority lev
   const expectedMessageCount = expectedLowCount + expectedMedCount;
   t.is(messages.length, expectedMessageCount);
 });
+
+test('handleSourceMappingEvent calls dispatch on messages in an EventSource event', async (t) => {
+  // EventSourceMapping input uses 'body' instead of 'Body'
+  const event = {
+    Records: [
+      {
+        body: createWorkflowMessage('test')
+      },
+      {
+        body: createWorkflowMessage('test')
+      }
+    ]
+  };
+  const output = await handleSourceMappingEvent(event);
+
+  const dispatchReturn = stubSFN().startExecution().promise();
+  output.forEach((o) => t.deepEqual(o, dispatchReturn));
```

Comment:
So what are we testing for deep equality here? Because `stubSFN().startExecution().promise();` returns `noop`, right?

Reply:
`t.is` also works, yeah. I guess I was future proofing, maybe? Good catch, I'll just make it `t.is`.

Reply:
I guess I'm also not totally clear on what the equal two values are that are being tested?

Reply:
I also replaced `noop` with `() => true` because it felt like checking for `undefined` could obscure errors in a way that returning an actual value does not.

Reply:
The basic idea is just to verify that the stub function was called.